include/zephyr/bluetooth/audio/audio.h

```diff
@@ -861,35 +987,37 @@ struct bt_audio_codec_qos_pref {
 	/** Preferred Transport Latency */
 	uint16_t latency;
 
-	/** @brief Minimum Presentation Delay in microseconds
+	/**
+	 * @brief Minimum Presentation Delay in microseconds
 	 *
-	 *  Unlike the other fields, this is not a preference but a minimum
-	 *  requirement.
+	 * Unlike the other fields, this is not a preference but a minimum requirement.
 	 *
-	 *  Value range 0 to @ref BT_AUDIO_PD_MAX, or @ref BT_AUDIO_PD_PREF_NONE
-	 *  to indicate no preference.
+	 * Value range 0 to @ref BT_AUDIO_PD_MAX, or @ref BT_AUDIO_PD_PREF_NONE
+	 * to indicate no preference.
 	 */
 	uint32_t pd_min;
 
-	/** @brief Maximum Presentation Delay
+	/**
+	 * @brief Maximum Presentation Delay
 	 *
-	 *  Unlike the other fields, this is not a preference but a maximum
-	 *  requirement.
+	 * Unlike the other fields, this is not a preference but a maximum requirement.
 	 *
-	 *  Value range 0 to @ref BT_AUDIO_PD_MAX, or @ref BT_AUDIO_PD_PREF_NONE
-	 *  to indicate no preference.
+	 * Value range 0 to @ref BT_AUDIO_PD_MAX, or @ref BT_AUDIO_PD_PREF_NONE
+	 * to indicate no preference.
 	 */
 	uint32_t pd_max;
 
-	/** @brief Preferred minimum Presentation Delay
+	/**
+	 * @brief Preferred minimum Presentation Delay
 	 *
-	 *  Value range 0 to @ref BT_AUDIO_PD_MAX.
+	 * Value range 0 to @ref BT_AUDIO_PD_MAX.
```

Comment:
This implies that `pd_ref_min` can be set lower than `pd_min`, right?

Reply:
Indeed. That is technically allowed today in the ASCS implementation, but not allowed in the spec. 

I'll create a separate PR to fix this, as it's more than just a documentation fix.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/75763